pyproject.toml

```diff
@@ -36,7 +36,7 @@ classifiers = [
     "Topic :: Documentation :: Sphinx",
 ]
 dependencies = [
-    "sphinx>=5.1.0",
+    "sphinx>=5.1.0,<9.0.0",
```

Comment:
This is a bit of a workaround for #82 , but in general I think it's a good idea not to allow newer Sphinx versions until we have seen they work, to prevent such issues again.

Reply:
In general upper-pinning is strongly advised against. We do it in Sphinx only as Docutils has historically had major breaking changes without warning.

A